docs/docs/tutorials/job-runtime.md

```diff
@@ -0,0 +1,44 @@
+# Job Runtime
+
+The job runtime plugin automatically collect runtime for every GitHub action, and present them in the pinned comment in pull requests.
+
+To enable this plugin, add a new task with the `job_runtime` plugin in your `.stoat/config.yaml` file:
+
+```yaml title=".stoat/config.yaml"
+---
+version: 1
+enabled: true
+tasks:
+  # Add a task with any arbitrary name.
+  job-runtime:
```

Comment:
I would call this something different like `build-time-history` or something just so it's clear which key represents the plugin itself.

Reply:
Will do.